src/modules/did.js

```diff
@@ -15,42 +15,73 @@ class DIDModule {
    * @constructor
    * @param {object} api - PolkadotJS API Reference
    */
-  constructor(api) {
+  constructor(api, signAndSend) {
     this.api = api;
     this.module = api.tx.didModule;
+    this.signAndSend = signAndSend;
   }
 
   /**
-   * Creates a new DID on the Dock chain.
+   * Creates transaction to create a new DID on the Dock chain.
    * @param {string} did - The new DID. Can be a full DID or hex identifier
    * @param {object} keyDetail - `KeyDetail` as expected by the Substrate node
    * @return {object} The extrinsic to sign and send.
    */
-  new(did, keyDetail) {
+  createNewTx(did, keyDetail) {
```

Comment:
I would have preferred not to have `create` as that's common for methods creating transactions but put the important part first like `newTx` or `updateKeyTx` but just expressing my opinion, not requiring a change